ci/common.rb

```diff
@@ -47,8 +47,10 @@ def section(name)
            -r requirements.txt\
            --cache-dir $PIP_CACHE\
            2>&1 >> $VOLATILE_DIR/ci.log)
+      sh %(PIP_OPTIONS="--cache-dir $PIP_CACHE" ./utils/pip-allow-failures.sh requirements-opt.txt\
```

Comment:
I don't know how it is possible, but an error is still displayed 
```bash
PIP_OPTIONS="--cache-dir $PIP_CACHE" ./utils/pip-allow-failures.sh requirements-opt.txt           2>&1 >> $VOLATILE_DIR/ci.log
    Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/fw/j46wyy091r38jhktdht1yprr0000gn/T/pip-build-Ms_ucm/psycopg2
pip install           -r requirements-test.txt           --cache-dir $PIP_CACHE            2>&1 >> $VOLATILE_DIR/ci.log
```
Maybe `pip` which always write to stdout...